### PR TITLE
Testing checks

### PR DESCRIPTION
**Pull request by catsmith**
This is the first step to testing. I have kept it in a separate branch for now you might be safer doing that too until we can prove it is all fine. I have fixed the problem in the last pull request which stopped the OTE code running without nodejs. I'm not sure it is the best solution but it works for now and will let us carry on with testing the wokflow for the tests. 

CI should run on push and pull_request. It seems to be working fine for me. 

To run locally you will need nodejs and then run `npm install` from the main directory. That should install everything you need to run the tests locally. `npm test` should then run the tests.

### CODE REVIEW

File: wce-ote/wce_tei.js [hunk 1]
```diff
@@ -4476,4 +4476,8 @@ var removeSpaceAfterLb=function ($node){
 	};
 
 
-module.exports = addArrows;
```
Comment (scribe777) [inline wce-ote/wce_tei.js]: I am wondering if we can just:
```
// for unit tests using node`
if (module) {
   module.exports = addArrows;
}
```
What do you think?

**Review by scribe777 — APPROVED** [review]
Looks good to me.  What do you think about adding the comment and if (module) block?  Actually, if you agree, I can do it in my next commit so you don't need to make any changes for this code release either way.

File: wce-ote/wce_tei.js (hunk 1 above)
Comment (catsmith) [reply wce-ote/wce_tei.js]: I tried that first, it didn't work. I still got errors and it refused to run the ote. I also tried testing for module in different ways and they all failed. The try catch is the only thing I could make work.

Comment (catsmith) [reply wce-ote/wce_tei.js]: I was wrong, it does load but it does give console errors. I know lots of packages don't care about those. If you think the error is okay then you can change it.

**Pull request merged**